src/services/tracing.ts

```diff
@@ -124,6 +124,18 @@ registerInstrumentations({
           requestHeaders: ["apollographql-client-name", "apollographql-client-version"],
         },
       },
+      applyCustomAttributesOnSpan: (span, _request, response) => {
```

Comment:
Where is this being used? And why is the span being passed in not the active one?